src/service/project.ts

```diff
@@ -300,3 +300,12 @@ export const getOwnLikeOfProjects = async (id: number) => {
     },
   });
 };
+
+export const createProjectView = async (userId: number, project_id: number) => {
+  await prisma.projectView.create({
+    data: {
+      user_id: userId,
+      project_id: project_id,
+    },
+  });
+};
```

Comment:
변수명은 camel case로 바꿔주세요 :D
```suggestion
export const createProjectView = async (userId: number, projectId: number) => {
  await prisma.projectView.create({
    data: {
      user_id: userId,
      project_id: projectId,
    },
  });
};
```